client/__test__/components/PostMessageForm.test.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import configureStore from 'redux-mock-store'
+import { PostMessageForm } from '../../components/postMessage/PostMessageForm';
+
+describe('Post message form component', () => {
+  const props = {
+    postMessage: jest.fn(() => Promise.resolve())
+  }
+  it('should render without crashing', () => {
+    const component = shallow(<PostMessageForm {...props}/>);
+    expect(component.node.type).toEqual('div');
+  });
+  it('should contain the method onChange', () => {
```

Comment:
From the assertion I see you do , you are testing the behaviour of the onChange method too, so I don't think your description should be _should contain onChange method_